src/python/WMComponent/AgentStatusWatcher/DrainStatusPoller.py

```diff
@@ -103,6 +104,11 @@ def updateAgentSpeedDrainConfig(self, thresholdsHit):
 
         return
 
+    def resetAgentSpeedDrainConfig(self):
+        if self.reqAuxDB.getWMAgentConfig(self.config.Agent.hostName).get("SpeedDrainMode"):
+            self.reqAuxDB.updateAgentConfig(self.config.Agent.hostName, "SpeedDrainMode", False)
```

Comment:
I am not resetting individual speed draining config since it will be updated when drain mode change to true.